tutorials-v5/brmesolve_phonon_interaction.md

```diff
@@ -0,0 +1,263 @@
+
+Instead, the Bloch-Redfield master equation allows for a direct connection between the quantum dynamics and the underlying physical interaction mechanism. Furthermore, the interaction strengths can be derived from first principles and can include complex power-dependences, such as those that exist in the quantum-dot-phonon interaction. Though we note that there are important non-Markovian effects which are now being investigated, e.g. in the paper <a href="https://journals.aps.org/prb/abstract/10.1103/PhysRevB.95.201305">Limits to coherent scattering and photon coalescence from solid-state quantum emitters</a>, Phys. Rev. B 95, 201305(R) (2017).
+
+
+### Problem parameters
+
+Note, we use units where $\hbar=1$.
+
+```python
+# pulse area
+n_Pi = 13
+
+# driving strengths
+Om_list = np.linspace(0.001, n_Pi, 80)
+# laser offsets in meV
+wd_list_e = np.array([-1, 0, 1])
+# laser offsets in angular frequency
+wd_list = wd_list_e * 1.5
+# simulation time with tmax ~ 2x FWHM
+tlist = np.linspace(0, 50, 40)
+
+# normalized Gaussian pulse shape, ~10ps long in energy
+t0 = 17 / (2 * np.sqrt(2 * np.log(2)))
+pulse_shape = "0.0867 * exp(-(t - 24) ** 2 / (2 * {0} ** 2))".format(t0)
+```
+
+### Setup the operators, Hamiltonian, and initial state
+
+```python
+# initial state
+psi0 = fock(2, 1)  # ground state
+
+# system's atomic lowering operator
+sm = sigmam()
+
+# Hamiltonian components
+H_S = -sm.dag() * sm  # self-energy, varies with drive frequency
+H_I = sm + sm.dag()
+
+# we ignore spontaneous emission since the pulse is much faster than
+# the decay time
+c_ops = []
+```
+
+Below, we define the terms specific to the Bloch-Redfield solver's system-environmental coupling. The quantum dot couples to acoustic phonons in its solid-state environment through a dispersive electron-phonon interaction of the form
+
+$$ H_\textrm{phonon}=\hbar J(\omega)\sigma^\dagger \sigma,$$
+
+where $J(\omega)$ is the spectral density of the coupling.
+
+```python
+# operator that couples the quantum dot to acoustic phonons
+a_op = sm.dag() * sm
+
+# This spectrum is a displaced gaussian multiplied by w^3, which
+# models coupling to LA phonons. The electron and hole
+# interactions contribute constructively.
+
+
+# fitting parameters ae/ah
+ah = 1.9e-9  # m
+ae = 3.5e-9  # m
+# GaAs material parameters
+De = 7
+Dh = -3.5
+v = 5110  # m/s
+rho_m = 5370  # kg/m^3
+# Other Constants
+hbar = 1.05457173e-34  # Js
+T = 4.2  # Kelvin, temperature
+
+# constants for temperature dependence
+t1 = 0.6582119
+t2 = 0.086173
+
+# General J factor
+J = (
```

Comment:
For v5 we should be able to avoid all of this somewhat ugly eval-ing and just create spectral coefficients from the strings and supply `w` and the other values (`ae`, `De`, etc) as arguments.